Week3/task08Vehicles.js

```diff
@@ -0,0 +1,27 @@
+const vehicleCodes = ['plane', 'car', 'motorbike','caravan','bike'];
+
+function vehicle(color, code, age) {
+    const valueOfColor = color;
+    const valueOfCode = vehicleCodes[code];
+    const valueOfAge = age <= 1 ? 'new' : 'used';
+    console.log('a '+valueOfColor, valueOfAge, valueOfCode)
+};
+vehicle("green", 3, 1);
+
+function advertisement() {
+    let strLast = vehicleCodes[vehicleCodes.length-1];
+    let arr1 = vehicleCodes.slice(0, [vehicleCodes.length-1]);
+    let str1 = arr1.toString();
+    let str2 = str1.replace(/,/g,', ');
+    console.log("Amazing Joe's Garage, we service "+str2+' and '+strLast+'.');
```

Comment:
Hmm.. I think this isn't complete. The requirement was to have the output like `"Amazing Joe's Garage, we service cars, motorbikes, caravans and bikes."` with plurals instead of singular values. Could you please have a look again?

Reply:
Yes, it wasn't completed. I finished and applied it now.